src/helpers/transactionHelpers.ts

```diff
@@ -0,0 +1,26 @@
+import { UserTransactionTypes } from '../components/wallet/UserTransactionsTable'
+export const txParty = (address: string, tx: UserTransactionTypes) => {
```

Comment:
As discussed before, add a test for any new helper you introduce. That's write tests for `txParty`, pls.